includes/class-edd-logging.php

```diff
@@ -334,10 +337,8 @@ public function update_log( $log_data = array(), $log_meta = array(), $log_id =
 
 		// Set log meta, if any
 		if ( 'edd_update_log' === $update_method && ! empty( $log_meta ) ) {
-			$log = edd_get_log( $log_id );
-
 			foreach ( (array) $log_meta as $key => $meta ) {
-				$log->update_meta( sanitize_key( $key ), $meta );
+				$meta_update( $log_id, sanitize_key( $key ), $meta );
```

Comment:
Same as above. 

1. Wrap foreach in `is_callable()`
2. Rename variable to `$update_meta_function`